plugins/inventory/nb_inventory.py

```diff
@@ -812,7 +812,16 @@ def get_role_for_rack(rack):
         self.racks_role_lookup = dict(map(get_role_for_rack, racks))
 
     def refresh_rack_groups_lookup(self):
-        url = self.api_endpoint + "/api/dcim/rack-groups/?limit=0"
+        if self.api_version == "2.11":
```

Comment:
```suggestion
        if self.api_version in ["2.11", "2.12"]:
```

Just to catch the two supported versions and this doesn't break when 2.12 is released.

Reply:
Good idea, but I did it a little more systemically - 2077f60973a0fe257570c5f149c1325ec21646ba